examples/tutorial/jupyter/execution/pandas_on_dask/test/test_notebooks.py

```diff
@@ -0,0 +1,186 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import os
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+
+test_dataset_path = "taxi.csv"
+ep = ExecutePreprocessor(timeout=600, kernel_name="python3")
+
+download_taxi_dataset = f"""import os
+import urllib.request
+if not os.path.exists("{test_dataset_path}"):
+    url_path = "https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv"
+    urllib.request.urlretrieve(url_path, "{test_dataset_path}")
+    """
+
+notebooks_dir = "examples/tutorial/jupyter/execution/pandas_on_dask/local"
+
+
+def _execute_notebook(notebook):
```

Comment:
See the comment about moving common stuff.

Reply:
Moved.